tools/copyblocks/README.md

```diff
@@ -1,9 +1,38 @@
 # Copyblocks
 
-This program can copy Mimir blocks between two GCS buckets. It is GCS specific and uses special calls to copy
-files between buckets directly, without download to a local system first.
+This program can copy Mimir blocks server-side between two buckets on the same object storage service provider.
+The currently supported services are Google Cloud Storage (GCS) and Azure Blob Storage (ABS).
 
-The copyblocks program can run a one-time copy job, or it can run continuously as a service that periodically checks to see if there are blocks to copy and if so runs the copy job.
+## Features
 
-You can configure it with a minimum block time range to avoid copying blocks that are too small.
-You can configure an allowlist and a blocklist of users to copy or not to copy.
+- Prevents copying blocks multiple times to the same destination bucket by uploading block marker files to the source bucket
+- Run as a one-time copy job, or run continuously with periodic checks (`--copy-period`)
```

Comment:
It's not super clear to me how `--copy-period` is linked to the idea of running this as a once-off or continuously - does this belong here?

Reply:
I rephrased this to try to make it clearer
> - Runs continuously with periodic checks when supplied a time duration with `--copy-period`, otherwise runs one check then exits.

This is the code it is referring to:
```go
	success := runCopy(ctx, cfg, logger, m)
	if cfg.copyPeriod <= 0 {
		if success {
			os.Exit(0)
		}
		os.Exit(1)
	}

	t := time.NewTicker(cfg.copyPeriod)
	defer t.Stop()

	for ctx.Err() == nil {
		select {
		case <-t.C:
			_ = runCopy(ctx, cfg, logger, m)
		case <-ctx.Done():
		}
	}
```
The copy period is nice during live migrations because you can run more frequent checks for new blocks.